cli.js

```diff
@@ -17,4 +17,14 @@ var cli = meow({
 	string: ['_']
 });
 
-cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite});
+try {
+	cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite}, function (err) {
```

Comment:
Yup, I usually do this to work around it:

```js
module.exports = function (src, dest, opts, cb) {
    if (!(Array.isArray(src) && src.length > 0) || !dest) {
     var err = new Error('`src` and `dest` required');
      err.noStack = true;
     throw err;
    }
```

(custom err prop)

Then check for it in cli.js

```js
if (err.noStack) {
  console.error(err.message);
  process.exit(1);
} else {
  throw err;
}
```